module/Application/view/application/index/index.phtml

```diff
@@ -26,7 +36,9 @@
                 Laminas Project is under active development. If you are
                 interested in following the development, you can check
                 <a href="https://getlaminas.org/blog/">Laminas dev blog</a>,
-                and <a href="https://github.com/issues?utf8=%E2%9C%93&amp;q=is:issue+org:laminas">Laminas issue tracker</a>
+                and <a
+                        href="https://github.com/issues?utf8=%E2%9C%93&amp;q=is:issue+org:laminas"
+                >Laminas issue tracker</a>
```

Comment:
I don't think it's more readable now. In a view script I would not set hard wrap at 80 or 100 columns.
The entire `a` element can be set on a new line but more is not needed here.